holoviews/plotting/mpl/path.py

```diff
@@ -103,3 +103,45 @@ def update_handles(self, axis, element, key, ranges=None):
         elif value is not None and np.isfinite(value):
             collection.set_array(np.array([value]*len(element.data)))
             collection.set_clim(ranges[vdim.name])
+
+
+class GeoMapPlot(PolygonPlot):
+    """
+    GeoMapPlot draws paths representing the region boundaries in the
+    supplied GeoMap. If the GeoMap has an associated value the color of
+    regions will be drawn from the supplied cmap, otherwise the supplied
+    facecolor will apply.
+    """
+
+    aspect = param.Parameter(default='equal', doc="""
+        GeoMap elements use an 'equal' aspect ratio by default but
+        may be set to an explicit aspect ratio or to 'square'.""")
+
+    bgcolor = param.Parameter(default='white', doc="""
+        Background color of the map.""")
+
+    frame = param.Boolean(default=False, doc="""
+        Whether to show the frame around the plot.""")
+
+    def initialize_plot(self, ranges=None):
+        # TODO: Most of this is copied & pasted from PolygonPlot.
+        #       How can we avoid the code duplication?!?
```

Comment:
I'm planning a refactor of matplotlib Element plots, which would eliminate most of the duplication. For now this is fine, I'll simply clean it up when that happens.